components/tidb_query_expr/src/impl_json.rs

```diff
@@ -204,9 +204,10 @@ fn quote(bytes: BytesRef) -> Result<Option<Bytes>> {
 
 #[rpn_fn(nullable)]
 #[inline]
-fn json_unquote(arg: Option<JsonRef>) -> Result<Option<Bytes>> {
+fn json_unquote(arg: Option<BytesRef>) -> Result<Option<Bytes>> {
```

Comment:
You may use non-nullable variant for this. Please remove nullable in rpn_fn macro and remove Option in both args and result, so that copr framework could automatically handle the case for null internally with better efficiency.

Reply:
Looks like all others functions in this file all use `nullable`. Maybe we open a new PR to fix all of them.

Reply:
As far as I could see, most functions in this file have non-nullable support, except for those accepting ScalarVlaueRef as parameter, which do not support non-nullable. Therefore, I suggest make this change in this PR.

Reply:
I agreed with @skyzh.

Reply:
done~